README.md

```diff
@@ -15,25 +15,135 @@ You can reach the maintainers of this project at:
+
+```sh
+export KUBECONFIG=</path/to/kubeconfig>
+```
+
+Override the docker image (optional)
+```sh
+export IMAGE_NAME=<your_own_image_name> # export IMAGE_NAME=quay.io/open-cluster-management/work:latest
+```
+
+And then deploy work webhook and work agent
+```
+make deploy
+```
+
+#### Deploy on two clusters
+
+Set environment variables. 
+
+- Hub and managed cluster share a kubeconfig file
+    ```sh
+    export KUBECONFIG=</path/to/kubeconfig>
+    export HUB_KUBECONFIG_CONTEXT=<hub-context-name>
+    export SPOKE_KUBECONFIG_CONTEXT=<spoke-context-name>
+    ```
+- Hub and managed cluster use different kubeconfig files.
+    ```sh
+    export HUB_KUBECONFIG=</path/to/hub_cluster/kubeconfig>
+    export SPOKE_KUBECONFIG=</path/to/managed_cluster/kubeconfig>
+    ```
+
+Set cluster ip if you are deploying on KIND clusters.
+```sh
+export CLUSTER_IP=<host_name/ip_address>:<port> # export CLUSTER_IP=hub-control-plane:6443
+```
+You can get the above information with command below.
+```sh
+kubectl --kubeconfig </path/to/hub_cluster/kubeconfig> -n kube-public get configmap cluster-info -o yaml
+```
+
+Override the docker image (optional)
+```sh
+export IMAGE_NAME=<your_own_image_name> # export IMAGE_NAME=quay.io/open-cluster-management/work:latest
+```
+
+And then deploy work webhook and work agent
+```
+make deploy
+```
+
+### What is next
+After a successful deployment, a namespace `cluster1` is created on the hub cluster. You are able to create ` ManifestWorks` in this namespace, and the workload will be distributed to the managed cluster and then be applied by the work agent.
+
+Create a manifest-work.yaml as shown in this example:
+```yaml
+apiVersion: work.open-cluster-management.io/v1
+kind: ManifestWork
+metadata:
+  name: hello-work
+  namespace: cluster1
+  labels:
+    app: hello
+spec:
+  workload:
+    manifests:
+      - apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: hello
+          namespace: default
+        spec:
+          selector:
+            matchLabels:
+              app: hello
+          template:
+            metadata:
+              labels:
+                app: hello
+            spec:
+              containers:
+                - name: hello
+                  image: quay.io/asmacdo/busybox
+                  command: ['sh', '-c', 'echo "Hello, Kubernetes!" && sleep 3600']
+```
+Apply the yaml file to the hub cluster.
+
+```
+kubectl --kubeconfig </path/to/hub_cluster/kubeconfig> apply -f manifest-work.yaml
+```
+
+Verify that the manifestwork resource was applied to the hub.
+
+```
+kubectl --kubeconfig </path/to/hub_cluster/kubeconfig> -n cluster1 get manifestwork/hello-work -o yaml
+```
+
+Check on the managed cluster and see the `Pod` has been deployed from the hub cluster.
+```
+kubectl --kubeconfig </path/to/managed_cluster/kubeconfig> -n default get pod
+NAME                     READY   STATUS    RESTARTS   AGE
+hello-64dd6fd586-rfrkf   1/1     Running   0          108s
+```
+
```

Comment:
I would add also some explanation on the status field in the work here. For instance, what is manifestConditions and what is workCondition. What does it mean by applied/availbale.

Reply:
Explanation on status added.